osidb/migrations/0092_migrate_cvss_to_flawcvss.py

```diff
@@ -60,6 +63,15 @@ def generate_flawcvss(apps):
 
 
 def forwards_func(apps, schema_editor):
+    set_user_acls(
```

Comment:
@Elkasitu I am thinking that the pieces of code using this construction will soon miss the private stuff

Reply:
For migrations it doesn't matter that much now, but I'll create an `ALL_GROUPS` and replace calls like this with that